appveyor.yml

```diff
@@ -58,6 +58,44 @@ on_failure:
 artifacts:
   - path: dist\*
 
+before_deploy:
+  # virtual env
+  - mkdir C:\venv
+  - C:\Python36-x64\Scripts\virtualenv C:\venv
+  - C:\venv\Scripts\activate
+  - python --version
+  - pip install -r requirements/deploy.txt
+  - pip install --editable .      # install wam, as needed by GB
+  # create standalone application
+  - gravitybee --src-dir src --sha file --with-latest --extra-data static --verbose
+  # prepare deployment envs
+  - .gravitybee\gravitybee-environs.bat
+  - set SATS_TAG=%GB_ENV_APP_VERSION%
+  - set SATS_BODY=* [%GB_ENV_APP_NAME% v%GB_ENV_APP_VERSION% CHANGELOG](https://github.com/plus3it/%GB_ENV_APP_NAME%/blob/%GB_ENV_APP_VERSION%/CHANGELOG.md)
+  - set SATS_REL_NAME=Release v%GB_ENV_APP_VERSION%
+  - SET # see all envs
+  # add artifacts
+  - ps: $root = Resolve-Path $Env:GB_ENV_STAGING_DIR; [IO.Directory]::GetFiles($root.Path, '*.*', 'AllDirectories') | % { Push-AppveyorArtifact $_ -FileName $_.Substring($root.Path.Length + 1) -DeploymentName to-publish }
+  - ps: ls
+  # deploy to github releases
+  - ps: >-
+      if($env:APPVEYOR_REPO_BRANCH -eq "master") {
+        satsuki --verbose
+      }
+# deploy to s3
+deploy:
+  - provider: Environment
+    name: wam-s3-dev
```

Comment:
where is it defined what this actually does?

Reply:
At AppVeyor website, at the top you'll see "ENVIRONMENTS" to right of "PROJECTS." Click that and you'll see the environments listed by name. Click on either one, then "SETTINGS" and you'll see the details (e.g., access key ID, bucket name, folder).

Reply:
Oh, that's at the global appveyor level, rather than the project level... is there some reason not to use the project settings? https://ci.appveyor.com/project/plus3it/watchmaker/settings/deployment

Reply:
You can't use the fancy environment feature in project settings.

Reply:
Never mind. You can do it all in project settings...

Reply:
looks like it could also be setup more similarly to travis:
```
deploy:
- provider: S3
  access_key_id: foo
  secret_access_key:
    secure: fhffELWjrOuP7+fHzE4lRQ==
  region: us-east-1
  bucket: baz
  on:
    branch: develop
```

Reply:
It's actually cleaner in the project settings because you can put the conditions there as well.